test/unit/specs/components/wallet/__snapshots__/LiTransaction.spec.js.snap

```diff
@@ -23,26 +23,28 @@ exports[`LiTransaction has the expected html structure 1`] = `
         class="tx-coin"
       >
         <div
-          class="value"
+          class="key"
         >
-          1,234.00
+          JBCOINS
         </div>
         <div
-          class="key"
+          class="value"
         >
-          jbcoins
+          1,234.00
         </div>
       </div>
     </div>
-    <div
-      class="tx-element tx-date"
-    >
-      a few seconds ago
-    </div>
-    <div
-      class="tx-element tx-address"
-    >
-      otherAddress
+    <div>
+      <div
+        class="tx-element tx-date"
+      >
+        Invalid date
```

Comment:
Is this expected?

Reply:
yes. remember the time zone issues i was having?